code/modules/cm_marines/marines_consoles.dm

```diff
@@ -700,13 +700,15 @@
 	idle_power_usage = 250
 	active_power_usage = 500
 	var/faction = FACTION_MARINE
+	/// Any extra factions this console should be tracking to.
+	var/list/extra_factions = list()
 	/// What type of /datum/crewmonitor this will create
 	var/crewmonitor_type = /datum/crewmonitor
 
 /obj/structure/machinery/computer/crew/Initialize()
 	. = ..()
 	if(!GLOB.crewmonitor[faction])
-		GLOB.crewmonitor[faction] = new crewmonitor_type(faction)
+		GLOB.crewmonitor[faction] = new crewmonitor_type(faction, extra_factions)
```

Comment:
doing this based on load order is just a bad idea, just either keep the crewmonitor datum on the monitor or key it based on the extra factions, eg

```
var/lookup_string = "[faction]-[json_encode(sort_list(extra_factions))"
if(!GLOB.crew_monitor[lookup_string]
	GLOB.crew_monitor[lookup_string] = new crew_monitor_type(faction, extra_factions)
```